fem/restriction.cpp

```diff
@@ -727,18 +855,36 @@ H1FaceRestriction::H1FaceRestriction(const FiniteElementSpace &fes,
 
    CheckFESpace(e_ordering);
 
-   ComputeScatterIndicesAndOffsets(e_ordering, type);
+   // Get the mapping from native DOF ordering to lexicographic ordering.
+   // For certain types of elements
```

Comment:
Incomplete comment?

Reply:
Fixed in f6efd6e56b4f8d60e0e8d80581f062f439618ee6